<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bluetooth</class>
 <widget class="QDialog" name="Bluetooth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>蓝牙连接</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: white;}
QPushButton {background-color: rgb(204, 204, 204);padding: 10px 12px 10px 12px;font: 19px 'Microsoft YaHei';letter-spacing: 1px;border: transparent;border-radius: 4px;}
QPushButton:pressed:hover {background-color: rgb(153, 153, 153);}
QPushButton:hover {background-color: rgb(230, 230, 230);}
QPushButton:disabled {background-color: rgb(204, 204, 204);color: rgb(122, 122, 122);}
QLabel{letter-spacing: 1px;}
QPushButton{letter-spacing: 2px;}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>670</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="search_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>搜索设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前无设备连接</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="attitude_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>4</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_Btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
